src/api/app/services/scm_status_reporter.rb

```diff
@@ -12,25 +13,30 @@ def call
     if github?
       github_client = Octokit::Client.new(access_token: @scm_token)
       # https://docs.github.com/en/rest/reference/repos#create-a-commit-status
-      github_client.create_status("#{@payload[:target_repository_full_name]}",
-                                  @payload[:commit_sha],
+      github_client.create_status("#{@event_subscription_payload[:target_repository_full_name]}",
```

Comment:
Are those `#{}` really needed?

Reply:
You're right, I don't think that they are needed.